sql/org_mozilla_firefox_derived/events_daily_v1/query.sql

```diff
@@ -36,9 +36,9 @@ joined AS (
   FROM
     events
   INNER JOIN
-    org_mozilla_firefox_derived.event_types_v1 event_types_v1
+    org_mozilla_firefox.event_types event_types_v1
```

Comment:
I guess this could be renamed to `event_types`

```suggestion
    org_mozilla_firefox.event_types event_types
```